liquibase-standard/src/main/java/liquibase/diff/output/changelog/DiffToChangeLog.java

```diff
@@ -340,6 +348,20 @@ public List<ChangeSet> generateChangeSets() {
         return changeSets;
     }
 
+    private void setReplaceIfExistsTrueIfApplicable(Change[] changes) {
+        if(changes !=null && diffOutputControl.isReplaceIfExistsSet()) {
+            for(int i=0; i < changes.length; i++) {
+                if (changes[i] instanceof CreateViewChange) {
```

Comment:
Here instead of the two if's you can just check if it is instanceof  ReplaceIfExists then call the method as both classes are implementing the interface, ie:

```
if (changes[i] instanceof ReplaceIfExists ) {
 ((ReplaceIfExists ) changes[i]).setReplaceIfExists(true);
}
```